### DISCUSSION TITLE
changing height of countrycode input

### DISCUSSION THREAD

**Discussion by AbdushukurUrinboev** [goveo/react-international-phone#72] [Q&A]
how can i change height of input of countrycode

**Comment by ybrusentsov** [comment]
@AbdushukurUrinboev
You can use the `countrySelectorStyleProps` prop for it:

```tsx
<PhoneInput
  value={phone}
  onChange={setPhone}
  defaultCountry="us"
  placeholder="Phone number"
  countrySelectorStyleProps={{
    dropdownStyleProps: {
      style: {
        maxHeight: "400px",
      },
      // className also supported
    },
  }}
/>
```

![image](https://github.com/goveo/react-international-phone/assets/25800848/8e5b15ba-e8b8-4761-9fb3-698fa3fb0924)


For some reason it uses `max-height` maybe it will be fixed to just `height` in next releases 🤔
